pkg/mkimage-raw-efi/install

```diff
@@ -357,6 +357,15 @@ if [ -f /config/device.cert.pem ] && [ -n "$REPORT" ]; then
    cat /config/device.cert.pem > "$REPORT/device.cert.pem"
 fi
 
+# If we have an inventory server then save report so we can push on first
+# boot
+INVENTORY_SERVER=$(tr ' ' '\012' < /proc/cmdline | sed -n '/eve_inventory_server=/s#eve_inventory_server=##p')
+if [ -n "$INVENTORY_SERVER" ]; then
+    mkdir /config/inventory
+    echo "$INVENTORY_SERVER" >/config/inventory/server
+    cp -rp "$REPORT" /config/inventory/
+fi
```

Comment:
I really don't like this to go into /config since we're still trying to make config as small (and immutable) as possible. Why can't we put this into /persist?

Reply:
> In general, this is a very welcome addition. However, my biggest concern is /config vs. /persist and also -- what would be the danger of trying to post it immediately and if that fails delaying it until the actual boot of EVE @eriknordmark ?

I'd love to change that, but the issues are that when the installer is running we do not have a /persist (that is created by storage-init on first boot), nor do we have a networking configured in the Linux kernel.

Reply:
While strictly speaking true, as we've discussed this shouldn't be too big of a deal to change that if needed. However, I'd agree -- for the proptotype/WIP it doesn't matter where it is.